test/verification_negative.json

```diff
@@ -12,7 +12,7 @@
         "signature": "0x7337bc2826c7678cd6bc84f5b3b236efc969b0451f9feca2328b1d3401b030c113f19bdba359ba3f52762c66e9147311fa95fe598a1a4ec9bb383a7b4e3874241b"
     },
     "domain binding": {
-        "domainBinding": "example.com",
+        "domain": "example.com",
```

Comment:
Duplicate field?

Do you know what `domainBinding` was supposed to be used for? I didn't know it was a thing when implementing SIWE in other languages

Reply:
I don't but based on other artifacts in the test data it struck me as intended to be `domain` and the tests were not supposed to be passing but the exception was being swallowed. I'll see if I can find the line in the diff

Reply:
Yes that's probably the case. Then you can remove one of the two